### DISCUSSION TITLE
firelomo：火狐的 flomo 快速发送扩展

### DISCUSSION THREAD

**Discussion by 7sDream** [flomoapp/3rd-party-tools#40] [分享创造]
flomo plus 没有提供火狐版本，就自己写了一个。

没写过浏览器扩展，也不会前端，这几天抽空临时学的。所以功能比较简单，只有快速发送到 flomo 的功能。

但是支持选中文字，选中链接，发送页面时使用不同的模板，可自定义。

项目地址：https://github.com/7sDream/firelomo

~由于火狐商店审核问题，暂未上线，可暂时通过 release 也页面下载。~

已上架：https://addons.mozilla.org/zh-CN/firefox/addon/firelomo/

**Comment by plidezus** [comment]
终于，有了第一版本！感谢~
